packages/ddp-client/livedata_connection_tests.js

```diff
@@ -95,6 +95,92 @@ Tinytest.add("livedata stub - receive data", function (test) {
   test.isUndefined(conn._updatesForUnknownStores[coll_name]);
 });
 
+Tinytest.add("livedata stub - buffering data", function (test) {
```

Comment:
I'm not sure I totally understand, but we can definitely add a dependency on `ecmascript`, certainly in the `onTest` section.